tutorials/dapt-curation/code/sources/github_repos.jsonl

```diff
@@ -0,0 +1,6 @@
+"google/skywater-pdk-libs-sky130_fd_sc_lp"
```

Comment:
Didn't we have way more repositories in this file? Did we remove for simplicity, or was there some other reason?

Asking because if someone wants to repro ChipNeMo data exactly, they'd need all the repos that was used for model training.

Reply:
@Maghoumi We cannot share the full data that ChipNeMo used because of legal permissions. So, we had to sample only a few datasets where permissions to share was easy to get.

Reply:
I thought the legal issue was with bundling all the data into the Google Drive share. Are you saying we can't include a link to the repositories, Wikipedia pages and arXiv papers?

Reply:
Yes exactly. Not all github repos and arxiv papers have permissions for us to share. So I had to change to a few sources where we have permissions.